tensorflow_addons/image/median_filter_2D_test.py

```diff
@@ -0,0 +1,95 @@
+import tensorflow as tf
+import median_filter_2D as md
```

Comment:
Could you make this module and the test module have a lowercase `d` so we don't break PEP

Reply:
Did not get this statement. please elaborate.

Reply:
Please rename the module to `median_filter_2d` upper case characters break pep:
https://www.python.org/dev/peps/pep-0008/#package-and-module-names

Reply:
fixed